docs/install/index.md

```diff
@@ -80,6 +80,12 @@ In case your distro doesn't have the package manager option listed above, procee
 ### Windows
 
 TBD in the version v0.0.8.
+- • Rust installation: [rust-lang.org/tools/install](https://www.rust-lang.org/tools/install)
```

Comment:
Can you move this to info to https://raphamorim.io/rio/install/#building ?

(By adding a new section with Windows after MacOS?)

You can skip the rust installation information (https://raphamorim.io/rio/install/#build-from-the-source)

Reply:
Thanks.